product-web-app/common/constants.ts

```diff
@@ -1,3 +1,6 @@
-const API_URL = `${process.env.API_URL}`;
+export const API_URL = `${process.env.API_URL}`;
 
-export default API_URL;
+export const API = {
+  JOB_ROLES: '/api/jobRoles/',
+  GET_JOB_ROLE: (id: number) => `api/jobRoles/${id}`,
```

Comment:
let's keep it lower case and dashes
```suggestion
  JOB_ROLES: '/api/job-roles/',
  GET_JOB_ROLE: (id: number) => `api/job-roles/${id}`,
```